### PR TITLE
Add fungible asset launchpad template (contract only)

### PR DESCRIPTION
**Pull request by 0xaptosj**
A launchpad for contract deployer to create FAs and anyone to mint FAs.

Only file needs review is the contract `/templates/fungible-asset-template/move/sources/fungible_asset_launchpad.move`

### CODE REVIEW

**Comment by 0xaptosj** [comment]
will remove `sh_scripts` later after i can use cli in ts

File: templates/fungible-asset-template/move/sources/fa_launchpad.move [hunk 1]
```diff
@@ -255,16 +276,23 @@ module launchpad_addr::fa_launchpad {
...
-            let obj = object::address_to_object<object::ObjectCore>(@launchpad_addr);
-            object::is_owner(obj, sender)
+    fun is_admin(config: &Config, sender: address): bool {
+        if (sender == config.admin_addr) {
+            true
         } else {
-            sender == config.admin_addr
+            if (object::is_object(@launchpad_addr)) {
+                let obj = object::address_to_object<object::ObjectCore>(@launchpad_addr);
+                object::is_owner(obj, sender) || sender == config.admin_addr
```
Comment (GotenJBZ) [inline templates/fungible-asset-template/move/sources/fa_launchpad.move:285]: the second condition is redundant

Comment (0xaptosj) [reply templates/fungible-asset-template/move/sources/fa_launchpad.move]: removed

**Comment by 0xaptosj** [comment]
close this one since we already break it into pieces and merge them

**Pull request closed without merging**